### PR TITLE
casablanca tma

### PR DESCRIPTION
**Pull request by obb12**
casablanca tma v1.0

### CODE REVIEW

**Comment by obb12** [comment]
wip

File: final/AF/MAR/GMMN35.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (AdamJCavanaugh) [inline final/AF/MAR/GMMN35.md:1]: Some formatting (heading, etc) might be nice?

File: final/AF/MAR/GMMN35.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Airspace from 2000-FL195
+
+On initial contact, arriving aircraft should be cleared for the arrival.
+
+
+
+
+
+---------------------------------------------------------
```
Comment (AdamJCavanaugh) [inline final/AF/MAR/GMMN35.md:24]: This whole section is shockingly huge. :)

**Comment by AdamJCavanaugh** [comment]
I'm having odd issues with aircraft under my control leaving the airspace (and becoming uncontrollable). Noticed between KOTAG and SKJ (with a plan on that approach), between BISMI and KOTAG (on the KOTAG approach) and around VARAS (with a plane on that approach).

**Comment by obb12** [comment]
> I'm having odd issues with aircraft under my control leaving the airspace (and becoming uncontrollable). Noticed between KOTAG and SKJ (with a plan on that approach), between BISMI and KOTAG (on the KOTAG approach) and around VARAS (with a plane on that approach).

Tma is now fixed

**Comment by AdamJCavanaugh** [comment]
I don't see new commits. Please make sure to push any updates you've made upstream so we can see them here, too?

**Comment by obb12** [comment]
it should be #578f4ea945052cb3719cecd3c91560c98df45b44

**Comment by AdamJCavanaugh** [comment]
Yeah, I see it now, sorry... it was added between me starting the review and commenting, I guess, so I missed it completely.

**Comment by AdamJCavanaugh** [comment]
It's odd to me that LACAJ is outside the airspace, so any aircraft using it would become uncontrollable? Similarly, I had a plane enter around LACAJ, directed them towards MOGBI, and they left the airspace and were uncontrollable until it reached MOGBI, which was... weird.

**Comment by obb12** [comment]
LACAJ and TUSOR are outside the tma 
before TUSOR arrival existed they used BISMI arrival

**Comment by AdamJCavanaugh** [comment]
Well, I'll leave it open for a bit to see if @jamesg31 wants to weigh in. If he doesn't or doesn't find any issues, then I'll merge it in. :)

**Comment by jamesg31** [comment]
The issue I am facing is that the start of all the STARs is on the border, LAJAC is even outside. It is impossible to actually clear the aircraft on the approach since by the time you can control them they have passed the start of the approach. The STARs look great, but its impossible to actually use them.

**Comment by obb12** [comment]
i can reroute SADIC/TUSOR to Bismi , NIKZO/FOBAC to CSD 
LAKAM to NASRO
LACAJ to GODPO
MOGBi to PIXOV
MABOG to VABNA 
that should close most of the issues

**Comment by obb12** [comment]
now it should be playable with all the arrivals

**Comment by jamesg31** [comment]
Now the issue is they will hold at some of the entry points. This is an annoying one! Maybe put the arrivals back how they where but just expand the border a little? Either way your going to need to sacrifice some realism just to get endless atc to work unfortunately.

**Comment by obb12** [comment]
updated

**Comment by obb12** [comment]
@jamesg31 do you think now its playble?

**Pull request merged**